src/ol/layer/Vector.js

```diff
@@ -52,6 +52,22 @@ import {createDefaultStyle, toFunction as toStyleFunction} from '../style/Style.
  */
 
 
+/**
+ * @enum {string}
+ * Render mode for vector layers:
+ *  * `'image'`: Vector layers are rendered as images. Great performance, but
+ *    point symbols and texts are always rotated with the view and pixels are
+ *    scaled during zoom animations.
+ *  * `'vector'`: Vector layers are rendered as vectors. Most accurate rendering
+ *    even during animations, but slower performance.
+ * @api
+ */
+export const RenderType = {
+  IMAGE: 'image',
+  VECTOR: 'vector'
+};
+
```

Comment:
I'm wondering if adding this was an accident.  We have this enum in `VectorRenderType` already.

Reply:
See #8845.